test/Microsoft.AspNetCore.Mvc.FunctionalTests/Microsoft.AspNetCore.Mvc.FunctionalTests.csproj

```diff
@@ -18,10 +17,13 @@
   <ItemGroup>
     <Compile Include="..\Microsoft.AspNetCore.Mvc.Formatters.Xml.Test\XmlAssert.cs" />
     <EmbeddedResource Include="compiler\resources\**\*" />
+    <Content Include="xunit.runner.json">
```

Comment:
Probably. We should do what we do elsewhere. 

`<None Include="xunit.runner.json" CopyToOutputDirectory="PreserveNewest" />`